conf/messages.en

```diff
@@ -349,6 +349,48 @@ trusteesInfo.5mld.details.paragraph2 = This is usually where the business is reg
+
+trustee.individual.5mld.countryOfNationality.title = What is the trustee’s country of nationality?
+trustee.individual.5mld.countryOfNationality.heading = What is {0}’s country of nationality?
+trustee.individual.5mld.countryOfNationality.checkYourAnswersLabel = What is {0}’s country of nationality?
+trustee.individual.5mld.countryOfNationality.error.required = Enter a country
+trustee.individual.5mld.countryOfNationality.error.length = The country of nationality must be 100 characters or less
+trustee.individual.5mld.countryOfNationality.error.invalidCharacters = The country of nationality must only include letters a to z, apostrophes, commas, full stops, hyphens, round brackets and spaces
+
+trustee.individual.5mld.countryOfResidenceYesNo.title = Do you know the trustee’s country of residence?
+trustee.individual.5mld.countryOfResidenceYesNo.heading = Do you know {0}’s country of residence?
+trustee.individual.5mld.countryOfResidenceYesNo.checkYourAnswersLabel = Do you know {0}’s country of residence?
+trustee.individual.5mld.countryOfResidenceYesNo.error.required = Select yes if you know the trustee’s country of residence
+trustee.individual.5mld.countryOfResidenceYesNo.hint = The country of residency is usually the country where the trustee lives and works most of the time during the tax year.
+
+trustee.individual.5mld.countryOfResidenceInTheUkYesNo.title = Is the trustee a UK residence?
+trustee.individual.5mld.countryOfResidenceInTheUkYesNo.heading = Is {0} a UK resident?
+trustee.individual.5mld.countryOfResidenceInTheUkYesNo.checkYourAnswersLabel = Is {0} a UK resident?
+trustee.individual.5mld.countryOfResidenceInTheUkYesNo.error.required = Select yes if the trustee has UK residency
+
+trustee.individual.5mld.countryOfResidence.title = What is the trustee’s country of residency?
+trustee.individual.5mld.countryOfResidence.heading = What is {0}’s country of residency?
+trustee.individual.5mld.countryOfResidence.checkYourAnswersLabel = What is {0}’s country of residency?
+trustee.individual.5mld.countryOfResidence.error.required = Enter a country
+trustee.individual.5mld.countryOfResidence.error.length = The country of residency must be 100 characters or less
+trustee.individual.5mld.countryOfResidence.error.invalidCharacters = The country of residency must only include letters a to z, apostrophes, commas, full stops, hyphens, round brackets and spaces
+
+trustee.individual.5mld.legallyIncapableYesNo.title = Does the trustee have mental capacity at the time of registration?
+trustee.individual.5mld.legallyIncapableYesNo.heading = Does {0} have mental capacity at the time of registration?
+trustee.individual.5mld.legallyIncapableYesNo.checkYourAnswersLabel = Does {0} have mental capacity at the time of registration?
+trustee.individual.5mld.legallyIncapableYesNo.error.required = Select yes if you know that the trustee’s has mental capacity
```

Comment:
Not sure what the prototype has but this doesn't make sense. I would suggest `Select yes if the the trustee has mental capacity at the time of registration`

Reply:
The prototype and spreadsheet with text have Enter yes, but I agree, Select yes makes more sense. I'll change it